README.md

```diff
@@ -4,13 +4,13 @@ This git repo contains the manual for Audacity
 ## Flow of Content
 
 * We write the Audacity Manual at https://alphamanual.audacityteam.org
-* We wget it and mangle it a little using our script, https://github.com/audacity/audacity/blob/master/scripts/mw2html_audacity/mw2html.py
+* We get it and mangle it a little using our script, https://github.com/audacity/audacity/blob/master/scripts/mw2html_audacity/mw2html.py
```

Comment:
[Nitpick/Proofreading/Question] I’m having trouble understanding what “mangle” means @Rajeta-Sarkar. Does it mean fix?
```suggestion
* We get it and mangle it a little using our script, https://github.com/audacity/audacity/blob/master/scripts/mw2html_audacity/mw2html.py
```